packages/sanes-chrome-extension/src/logic/persona/personamanager.ts

```diff
@@ -0,0 +1,44 @@
+import { Persona } from './persona';
+
+export class PersonaManager {
+  private static instance: Persona | undefined;
+
+  /**
+   * Creates and stores a Persona instance
+   */
+  public static async create(fixedMnemonic?: string): Promise<Persona> {
+    const newInstance = await Persona.create(fixedMnemonic);
+    PersonaManager.instance = newInstance;
+    return newInstance;
+  }
+
+  /**
+   * Gets the global Persona instance
+   *
+   * Throws if instance does not exist, which indicates a bug in the app life cycle.
+   */
+  public static get(): Persona {
+    if (!PersonaManager.instance) {
+      throw new Error('Persona instance does not exist');
+    }
+    return PersonaManager.instance;
+  }
+
+  /**
+   * Gets the global Persona instance when there is one
+   */
+  public static getWhenCreated(): Persona | undefined {
```

Comment:
React views & components won't access Persona directly, they will access the info in a different way. Not directly because we lose updates on the data, and react by design should:
1. Not have an idea where data comes from, meaning react only cares about received props.
2. Should be dynamic in order to recompose the react tree on data updates (we do not want mutations)

So this method should be removed. For now, if you want, wrap access to Persona in views using try/catch.